controller/controller_server/addons_server.py

```diff
@@ -0,0 +1,160 @@
+import grpc
+
+from controller.array_action import errors as array_errors
+from controller.array_action.config import REPLICATION_DEFAULT_COPY_TYPE
+from controller.array_action.storage_agent import get_agent
+from controller.common.csi_logger import get_stdout_logger
+from controller.common.utils import set_current_thread_name
+from controller.controller_server import config, utils
+from controller.controller_server.exception_handler import handle_common_exceptions, build_error_response
+from controller.csi_general import replication_pb2 as pb2
+from controller.csi_general import replication_pb2_grpc as pb2_grpc
+
+logger = get_stdout_logger()
+
+
+class ReplicationControllerServicer(pb2_grpc.ControllerServicer):
+    @handle_common_exceptions(pb2.EnableVolumeReplicationResponse)
+    def EnableVolumeReplication(self, request, context):
+        set_current_thread_name(request.volume_id)
+        logger.info("EnableVolumeReplication")
+        utils.validate_addons_request(request)
+
+        volume_id_info = utils.get_volume_id_info(request.volume_id)
+        volume_id = volume_id_info.object_id
```

Comment:
it starting to be confusing, can we consider changing object_id to wwn or something like that?

Reply:
not sure why this PR is the "start" of the confusion, it was like this before.
but in general, I agree we can consider `wwn`/`wwid`/`uuid`/etc... across the project (need to check what is the correct terminology here)

Reply:
sounds good